docs/other/windows.md

```diff
@@ -44,6 +44,27 @@ You can use the [vagrant-winnfsd](https://github.com/GM-Alex/vagrant-winnfsd) pl
 
 Another option for the more adventurous is to manually install and configure WinNFSD, and manually mount the shares within your VM. This requires a bit more work, but could be more stable on Windows; see this blog post for more details: [Windows + Vagrant + WinNFSD without file update problems](https://hollyit.net/blog/windowsvagrantwinnfsd-without-file-update-problems).
 
+There is a fork of vagrant-winnfsd that add logging and debug https://github.com/GuyPaddock/vagrant-winnfsd, replace in your .vagrant.d\gems\gems folder to test it.
```

Comment:
This sentence would read a little better:

GuyPaddock's [fork of vagrant-winnfsd](https://github.com/GuyPaddock/vagrant-winnfsd) adds logging and debug messages. You can replace the `vagrant-winnfsd` gem inside `.vagrant.d\gems\gems` to use it instead.